SystemPreferences/README.md

```diff
@@ -51,6 +51,9 @@ $ killall Dock
 
 ## User Defaults
 - Enable repeating keys by pressing and holding down keys: `defaults write NSGlobalDomain ApplePressAndHoldEnabled -bool false` (and restart any app that you need to repeat keys in)
+- Change the default folder for screenshots
+    - Open the terminal and make a new folder somewhere on your computer: `mkdir /path/`
```

Comment:
I think it would be more explicit if this line could be changed to something like:
> Open the terminal and create the folder where you would like to store your screenshots: `mkdir /my/custom/path/`

And then on the next line `/path/` would have to change to `/my/custom/path/` as well.

I feel like a longer example path will make the reader more aware of the fact that they are creating a new path and also that they have to use the _same_ path in the command below. `/path/` is a bit short and it could be overlooked.

Reply:
Thank you! I agree, usability counts. Changed `/path/` to `/path/to/screenshots/`.